src/main/java/seedu/sellsavvy/logic/commands/personcommands/EditPersonCommand.java

```diff
@@ -55,6 +55,12 @@ public class EditPersonCommand extends Command {
     public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
     public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book.";
+    public static final String MESSAGE_SIMILAR_TAGS_WARNING = "Note: "
+            + "This customer has 2 or more similar tags after editing tags, "
+            + "verify if this is a mistake.\n";
+    public static final String MESSAGE_SIMILAR_NAME_WARNING = "Note: "
+            + "A person with similar name already exists in the address book, "
+            + "verify if this is a mistake.\n";
```

Comment:
Messages presently do not use breakline, check if this is valid without `\n`?

Reply:
Negligible effect anyway.

Reply:
it need to break line so because there can be multiple warnings and we need ensure it is at different line if there is a warning already